stable/velero/templates/configmaps.yaml

```diff
@@ -0,0 +1,6 @@
+{{- range .Values.configMaps }}
+---
+apiVersion: v1
+kind: ConfigMap
+{{ toYaml . }}
```

Comment:
I'd suggest to keep a generic and static structure for the `metadata`  and `data` properties such as:

```
metadata:
  name: {{ .name }}
  labels:  {{ .labels }}
data:
 {{ .data }}
```

Reply:
This was the way i've done it in my draft version. I was not sure about that so i changed it. The problem i see with that is that if user want to add annotations, or other things, the chart need to be updated.

Before applying your suggestion, i will check other charts to see if there exists a common pattern for that generic way of adding resources.

Thank for your feedback!

Reply:
> The problem i see with that is that if user want to add annotations, or other things, the chart need to be updated.

Good point. We could also add an annotation's field too:
```
{{- with .Values.xxx.annotations }}
  annotations:
{{ toYaml . | indent 4 }}
{{- end }}
```

Reply:
Otherwise, just skip the annotations field and let others add it, if required.

Reply:
I've seen how it's done in ```templates/schedules.yaml```

What do you think of something like that (not tested yet) :

```
{{- range .Values.configMaps }}
apiVersion: v1
kind: ConfigMap
metadata:
  name: {{ include "velero.fullname" $ }}-{{ .name }}
  labels:
    app.kubernetes.io/name: {{ include "velero.name" $ }}
    app.kubernetes.io/instance: {{ $Release.Name }}
    app.kubernetes.io/managed-by: {{ $Release.Service }}
    helm.sh/chart: {{ include "velero.chart" $ }}
  {{- with .labels }}
    {{- toYaml .labels | nindent 4 }}
  {{- end }}
data:
  {{- toYaml .data | nindent 2 }}
---
{{- end }}
```

The good things are the standard labels and a name that is now build on the release name

Reply:
I've finally implemented, and tested, something like the solution i suggested.